Unix/doc/setup-ntlm-omi.md

```diff
@@ -180,6 +180,17 @@ On the command line:
 
 The most secure method for providing omiserver credentials is via winbind. There may be reasons why this would not be 
 practical for a particular implementation, in which case it is possible to define a credential file, using the same format as
-described for the client.  If this method is used, the location of the file is specified in the omiserver.conf file.
+described for the client.  If this method is used, use of the file is specified by setting the option useNtlmCredsFile=true.
+The file is kept in a standard place ( /etc/opt/omi/creds/ntlm in the release version, output/etc/creds for configure --dev )
```

Comment:
Nice too keep the docs up to date, good job. That said, I think this is wrong ... (needs to be read in from omiserver.conf).